src/axom/quest/tests/quest_sampling_shaper.cpp

```diff
@@ -1038,6 +1052,204 @@ units: cm
   }
 }
 
+TEST_F(SamplingShaperTest2D, check_underscores)
```

Comment:
This test checks that we can use material names that contain underscores. 
Previously, a bug in how the material names were parsed only used the substring up to the first underscore.